### PR TITLE
Remove references to JIRA in the README

### PR DESCRIPTION
**Pull request by 0xTim**
JIRA is no longer used for Swift, references updated to GitHub

### CODE REVIEW

**Review by mportiz08 — COMMENTED** [review]
I think there might also be some references to JIRA in the `CONTRIBUTING.md` file. Would you mind updating those too?

Thanks for catching this!

**Comment by 0xTim** [comment]
@mportiz08 done!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ from the community.
 ### Contributing Code and Documentation
 
 Before contributing code or documentation to Swift-DocC-Render
-we encourage you to first create an issue on [Swift JIRA](https://bugs.swift.org/)
+we encourage you to first create an issue on [GitHub](https://github.com/apple/swift-docc-render/issues/new)
```
Comment (mportiz08) [inline CONTRIBUTING.md:38]:
```suggestion
we encourage you to first create an issue on [GitHub](https://github.com/apple/swift-docc-render/issues/new/choose)
```

**Review by mportiz08 — COMMENTED** [review]
Awesome, thanks!

I made some minor suggestions to the URLs to utilize existing GitHub Issues templates that we already have.

**Review by mportiz08 — APPROVED** [review]

**Pull request merged**